challenge-03/README.md

```diff
@@ -35,103 +46,122 @@ valor dessa propriedade a quantidade passada por parâmetro;
 - Ele deverá modificar o valor da propriedade `andando` para o valor
 booleano que representa "verdadeiro";
 */
-?
+	pessoa.andar = function(metrosCaminhados) {
+		pessoa.caminhouQuantosMetros += metrosCaminhados;
+		if (pessoa.andando === false) {
```

Comment:
-1
Aqui não precisa do `if`. Sempre deve setar o valor `pessoa.andando` para `true` quando esse método for utilizado ;)
Entende o porquê?